lighthouse-core/audits/errors-in-console.js

```diff
@@ -81,14 +81,15 @@ class ErrorLogs extends Audit {
     /** @type {AuditOptions} */
     const auditOptions = context.options;
 
-    /** @type {Array<{source: string, description: string|undefined, url: string|undefined}>} */
+    /** @type {Array<{source: string, description: string|undefined, sourceLocation: LH.Audit.Details.SourceLocationValue|undefined}>} */
     const consoleRows = artifacts.ConsoleMessages
       .filter(item => item.level === 'error')
       .map(item => {
         return {
           source: item.source,
           description: item.text,
           url: item.url,
```

Comment:
wondering if `url` should stick around and be removed in v8 (ditto for all these other audits), or are we good with removing the old properties immediately?

Reply:
I don't have a strong opinion on whether we should keep the `url` field around until v8. However, if you decide to leave it for now please add a TODO to remove it later.

Reply:
I like leaving it here too 👍